pkg/service/sos/objectstorage.go

```diff
@@ -183,3 +192,22 @@ func addOrgAndNamespaceToBucket(ctx context.Context, buckets exoscalev1.BucketLi
 	}
 	return bucketDetails
 }
+
+func fetchNamespaceWithOrganizationMap(ctx context.Context, k8sclient client.Client) (map[string]string, error) {
+	log := ctrl.LoggerFrom(ctx)
+	list := &corev1.NamespaceList{}
+	if err := k8sclient.List(ctx, list, client.HasLabels{service.OrganizationLabel}); err != nil {
```

Comment:
I would update how we fetch namespaces here as well like we did for dbaas. Just remove the `hasLabels`.